### ISSUE TITLE
Wrong filename on healthcheck HELP message for assertConfigFiles.

### ISSUE THREAD

**Issue by VFS** [passbolt/passbolt_api#362]
## Wrong filename on healthcheck HELP message for assertConfigFiles.
* Passbolt Version: 2.13.5
* Platform and Target:
-- Operating system: Baseimage from https://hub.docker.com/r/passbolt/passbolt/
-- PHP: 7.3.21
-- Web server: nginx/1.14.2
-- Database server: MariaDB - 10.5.4
-- etc.: Configured using the latest docker image 

### What you did
I ran `su -s /bin/bash -c "/var/www/passbolt/bin/cake passbolt healthcheck" www-data`

### What happened
Looking through the health check, I saw there was a warning for a missing file. While attempting to follow the "HELP" block, I fount out the file has a different name.
```shell
...
Config files

 [PASS] The application config file is present
 [WARN] The passbolt config file is missing in /var/www/passbolt/config/
  [HELP] Copy /var/www/passbolt/config/passbolt.php.default to /var/www/passbolt/config/passbolt.php
  [HELP] The passbolt config file is not required if passbolt is configured with environment variables
...
```

But the file is named `passbolt.default.php`:

```
root@passbolt_api_server:/var/www/passbolt/config# ls
Migrations  app.default.php  bootstrap.php	bootstrap_plugins.php  file_storage.php  passbolt.default.php  requirements.php  schema
Seeds	    app.php	     bootstrap_cli.php	default.php	       gpg		 paths.php	       routes.php	 version.php
```

### What you expected to happen
The default config file is named `passbolt.default.php` instead of `passbolt.php.default`, as can be seen at https://github.com/passbolt/passbolt_api/blob/master/config/passbolt.default.php

Should I just create a pull request updating that string at [src/Shell/Task/HealthcheckTask.php](https://github.com/passbolt/passbolt_api/blob/d11be29ef9f3273674dd7350c108f2902cf29e7a/src/Shell/Task/HealthcheckTask.php#L273) , or are there other required steps? Thanks!

**Comment by cedricalfonsi** [comment]
Hello @VFS,

Thank you for the report.
You can definitely create a PR, it will be very much appreciated.

**Comment by stripthis** [comment]
@cedricalfonsi  I already fixed that on common branch. @VFS no need for a PR we already have a fix incoming. Thanks for the report.

**Comment by stripthis** [comment]
Ref. https://github.com/passbolt/passbolt_api/commit/d50618a3fc7f85e4de5d1d2a25be1667bfdabaf7

**Comment by VFS** [comment]
@stripthis Awesome! Should I wait for it to be merged with Master to close this issue?

**Comment by stripthis** [comment]
Fixed with v3.0.0.
